projects/dsp-ui/src/lib/viewer/operations/display-edit/display-edit.component.spec.ts

```diff
@@ -324,14 +322,11 @@ describe('DisplayEditComponent', () => {
         {
             provide: MatDialogRef,
             useValue: {}
-        },
-        ValueService,
```

Comment:
@mdelez I think there is no need to provide the service here since it is provided in root anyway. 
Importing `ReactiveFormsModule` fixes the injector problem with `FormBuilder`.

Reply:
However, I am not really sure what the best way is. Maybe we should spend some time on https://dasch.myjetbrains.com/youtrack/issue/DSP-98

Reply:
I'm currently writing unit tests for the PR mentioned above and the imports/injections (in test host components) for the FormBuilder are quite confusing. I think we should spend some time on what exactly needs to be imported in the unit tests that you don't get an injector error.

Reply:
That is an excellent idea. Maybe we could try to work on https://dasch.myjetbrains.com/youtrack/issue/DSP-98 soon.